frame/nomination-pools/src/mock.rs

```diff
@@ -175,7 +186,7 @@ impl pools::Config for Runtime {
 	type PalletId = PoolsPalletId;
 	type MaxMetadataLen = MaxMetadataLen;
 	type MaxUnbonding = MaxUnbonding;
-	type MinPointsToBalance = MinPointsToBalance;
+	type MinPointsToBalance = frame_support::traits::ConstU32<10>;
```

Comment:
is there a reason this is `frame_support::traits::ConstU32<10>` but something of the same type like `MaxMetadataLen` is still a parameter type?

Reply:
No, just being forgotten. The ones that will rarely change should be `ConstU32`.